incubator/drone/README.md

```diff
@@ -0,0 +1,65 @@
+# Drone.io
+
+[Drone](http://readme.drone.io/) is a Continuous Integration platform built on container technology. Every build is executed inside an ephemeral Docker container, giving developers complete control over their build environment with guaranteed isolation.
+
+## Introduction
+
+This chart stands up a Drone. This includes:
+
+- A [Drone Server](http://readme.drone.io/admin/installation-guide/) Pod
+- A [Drone Agent](http://readme.drone.io/admin/installation-guide/) Pod
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- The ability to point a DNS entry or URL at your Drone install
+
+## Installing the Chart
+
+To install the chart with the release name `my-release` run:
+
+```bash
+$ helm install --name my-release \
+    --set remote.gogs.enabled=true,remote.gogs.url=http://your-gogs-domain.com/ incubator/drone
+```
+
+Note that you _must_ pass an remote settings, or you'll end up with a non-functioning release.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+Refer to [values.yaml](values.yaml) for the full run-down on defaults. These are a mixture of Kubernetes and drone.io directives.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
+    --set remote.gogs.enabled=true,remote.gogs.url=http://your-gogs-domain.com/ \
+    incubator/drone
+```
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
```

Comment:
this is all very basic helm usage that the tutorials go over. I don't think this should be documented here.